### PR TITLE
[backend] Refine report generate service

### PR DESCRIPTION
**Pull request by mjx20045912**
Refine report generate service

### CODE REVIEW

File: backend/src/main/java/heartbeat/service/report/ReportService.java [hunk 1]
```diff
@@ -14,15 +18,27 @@ public class ReportService {
...
 		}
-		return csvFileGenerator.getDataFromCSV(reportType, csvTimestamp);
+		return csvFileGenerator.getDataFromCSV(reportDataType, csvTimestamp);
 	}
 
 	private boolean isExpiredTimeStamp(long timeStamp) {
 		return timeStamp < System.currentTimeMillis() - EXPORT_CSV_VALIDITY_TIME;
 	}
 
+	public void generateReportByType(GenerateReportRequest request, ReportType metricType) {
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/service/report/ReportService.java:34]: ReportType-> MetricType

File: backend/src/main/java/heartbeat/controller/report/dto/request/ReportType.java [hunk 2]
```diff
@@ -2,23 +2,18 @@
 
 public enum ReportType {
 
-	METRIC("metric"), PIPELINE("pipeline"), BOARD("board");
+	BOARD("board"), DORA("dora");
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/controller/report/dto/request/ReportType.java:5]: ReportType is defined `MetricType` in main branch

**Review by guzhongren — COMMENTED** [review]
rebase的时候感觉是个大change

File: backend/src/main/java/heartbeat/controller/report/dto/request/ReportType.java (hunk 2 above)
Comment (mjx20045912) [reply backend/src/main/java/heartbeat/controller/report/dto/request/ReportType.java:5]: We refine the domain

**Review by mikeyangyun — APPROVED** [review]

**Pull request merged**